### PR TITLE
Live kill fixes

### PR DESCRIPTION
**Pull request by phughk**
Review changes for statments live+kill and integration WS live/kill (no changes there).

- Change LQ description to be a bit clearer how it works
- Changed KILL to clarify that it allows params

### CODE REVIEW

**Comment by phughk** [comment]
Live Changes:
Screenshot 2023-08-09 at 11 52 49
b-101c-49f1-b47d-50aab6382ff7">

File: app/templates/docs/surrealql/statements/kill.hbs [hunk 1]
```diff
@@ -26,6 +30,9 @@
 	<h4>Basic usage</h4>
 	<p>The <code>KILL</code> statement expects the UUID of a running live query to be passed.</p>
 	<Code @name="docs/surrealql/statements/kill/simple.surql" />
+
+	<p>The <code>KILL</code> statement will also allow for parameters to be used.</p>
```
Comment (kearfy) [inline app/templates/docs/surrealql/statements/kill.hbs:34]: will also allow for -> also allows for

**Review by kearfy — CHANGES_REQUESTED** [review]

File: app/templates/docs/surrealql/statements/kill.hbs (hunk 1 above)
Comment (phughk) [inline app/templates/docs/surrealql/statements/kill.hbs:34]:
```suggestion
	<p>The <code>KILL</code> statement also allows for parameters to be used.</p>
```

**Review by kearfy — APPROVED** [review]
LGTM!

**Pull request merged**